src/utils/AuthApi.test.ts

```diff
@@ -0,0 +1,49 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { useAuthenticationApi } from './AuthApi';
+import { ApiResponse } from './UseAxios';
+import MockAdapter from 'axios-mock-adapter';
+import axios from 'axios';
+
+const email = 'testEmail@email.com';
+const password = 'TestPassword';
+
+const httpHeaders = new Headers({
+  'Content-Type': 'json',
+  'Project-Name': 'sdc',
+});
+
+const getRequestData = {
+  user: { email, password },
+};
+
+const baseUrl = `${process.env.REACT_APP_API_URL}`;
+
+describe('useAuthenticationApi', () => {
```

Comment:
These tests look fantastic so far, would be nice to having a loading and failure test as well.

Reply:
Makes sense. I've added loading and failure tests for all the API wrappers.